storybook/stories/Examples/AreaChart.stories.tsx

```diff
@@ -66,6 +66,55 @@ export const StackedAreaChart = {
   },
 };
 
+export const StackedAreaChartWithMouseEvents = {
+  render: () => {
+    const handleContextMenu = useCallback((state: any, event: any) => {
+      event.preventDefault();
+      // eslint-disable-next-line no-console
+      console.log('On context menu', { state, event });
+    }, []);
+
+    const handleDoubleClick = useCallback((state: any, event: any) => {
+      event.preventDefault();
+      // eslint-disable-next-line no-console
+      console.log('On double click', { state, event });
+    }, []);
+
+    const handleClick = useCallback((state: any, event: any) => {
```

Comment:
Any particular reason useCallback is needed?

Reply:
Just a bad habit here. Will remove it.

Reply:
It's not a bad thing, just want to keep it simple for the sake of the examples